### DISCUSSION TITLE
DSS_NIK plotly.express error

### DISCUSSION THREAD

**Discussion by ammaralwandi97** [teamalgoritma/community#820] [Q&A] [answered]
plotly express error tidak bisa menampilkan grafik. padahal sudah dinstall package nya. 
![image](https://github.com/teamalgoritma/community/assets/89758039/52610fb8-e648-43b1-8922-9876d4566ed4)
![image](https://github.com/teamalgoritma/community/assets/89758039/a0d74dd5-6e52-4ac4-83c3-fb2b2f624a37)

**Comment by kinantytasyao** [comment, marked as answer]
Dear @ammaralwandi97 

Error yang terlampir menunjukkan bahwa nbformat belum terinstall. Berikut adalah code untuk mengintall nbformat di terminal cmd:

`pip install nbformat`

Demikian yang dapat kami sampaikan, semoga membantu :)

Jika ternyata jawaban ini menyelesaikan permasalahan Anda silakan mark as answer.
